sdk/vc-data/src/data/accountOwnership/v1.test.ts

```diff
@@ -0,0 +1,158 @@
+import { getAccountOwnershipV1Context, VCAccountOwnershipV1, VCSAccountOwnershipV1 } from './v1'
+import { expandVC } from '../../testUtil.test'
+
+describe('AccountOwnershipV1', () => {
+  it('expands correctly', async () => {
+    expect.assertions(1)
+
+    const expanded = await expandVC<VCAccountOwnershipV1, VCSAccountOwnershipV1>({
+      type: 'AccountOwnershipV1',
+      data: {
+        '@type': 'AccountOwnership',
+        accountName: 'bobbelcher',
+        accountType: 'github',
+        metaData: {
+          id: 42,
+          login: 'bobbelcher',
+          url: 'https:github.com/bobbelcher',
+          followers: 5000,
+          following: 200,
+          twitterUsername: 'bobbelche-twits',
+          email: 'bobbelcher@gmail.com',
+          name: 'Bob Belcher',
+          repos: [
+            {
+              id: 4242,
+              fullName: 'bobbelches_popularRepo',
+              ownerLogin: 'bobbelcher',
+              url: 'https:github.com/bobbelches_popularRepo',
+              forksCount: 100,
+              stargazersCount: 2000,
+              watchersCount: 300,
+            },
+          ],
+        },
+      },
+      context: getAccountOwnershipV1Context(),
+    })
+
+    expect(expanded).toMatchInlineSnapshot(`
+      Object {
+        "@id": "urn:uuid:9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6d",
+        "@type": Array [
+          "https://www.w3.org/2018/credentials#VerifiableCredential",
+          "https://schema.affinity-project.org/AccountOwnershipV1",
+        ],
+        "https://www.w3.org/2018/credentials#credentialSubject": Array [
+          Object {
+            "https://schema.affinity-project.org/data": Array [
+              Object {
+                "@type": Array [
+                  "https://schema.affinity-project.org/AccountOwnership",
+                ],
+                "https://schema.affinity-project.org/accountName": Array [
```

Comment:
Some of this fields may use `schema.org` instead of our own type maybe? For example this could be an alias for `https://schema.org/name` or we can just name it as `name`. Same goes for email etc.